### ISSUE TITLE
requirement: referencing WCAG 2 or not?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-wcag-videos#43]
Although the videos are being planned to go along with specific SC, afaik, the videos themselves are not planned to mention the SC? What are pros and cons to that?

Considerations:
* It would be good for them to be smoothly usable in multiple places, e.g., Understanding WCAG 2 SC, Content Usable, WCAG 3, courses and presentations, etc.
* When the video is seen outside of Understanding WCAG 2, what are the benefits of knowing that it is related to specific WCAG 2 SC (e.g., good to know that relationship when watching it in different context, e.g., external training course, Content Usable, or WCAG 3)?
* What would be the cons (e.g., in WCAG 3 might seem outdated)?

As this *keeps* coming up, I think we should explicitly address it in at least: 
* https://www.w3.org/WAI/EO/wiki/Video-Based_Resources/WCAG_Requirements* 

and probably
* https://github.com/w3c/wai-wcag-videos/blob/master/README.md

**Comment by nitedog** [comment]
Agree that this keeps coming up and needs to be clarified.

We probably do want a URL at the end of the videos for when they are used outside the Understanding docs. This could be fairly generic, like one of the following:
 - https://www.w3.org/WAI/standards-guidelines/
 - https://www.w3.org/WAI/standards-guidelines/wcag/
 - https://www.w3.org/WAI/

**Comment by shawna-slh** [comment]
Could we (EOWG+) **soon** agree on and document if:
* video content will not include the SC number or be limited to a specific SC or guideline directly
* video file name will not include the SC number or be limited to a specific SC or guideline directly
* some video content or file names might use the handle or other words from the SC, yet those would still be relevant outside of WCAG

Then add those decisions and points to the links in first comment, so hopefully people will see them and those that don't we can point them there. (to avoid answering the same question over and over)

We don't need to decide on URI at the end for quite a while. :-)
[new URI issue](https://github.com/w3c/wai-website/issues/250)

**Comment by nitedog** [comment]
> -    video content will not include the SC number or be limited to a specific SC or guideline directly
> -    video file name will not include the SC number or be limited to a specific SC or guideline directly
> -    some video content or file names might use the handle or other words from the SC, yet those would still be relevant outside of WCAG

The statements "or be limited to a specific SC or guideline directly" and "would still be relevant outside of WCAG" are somewhat ambiguous. My understanding of the [requirements that EOWG agreed on](https://www.w3.org/WAI/EO/wiki/Video-Based_Resources/WCAG_Requirements) is that the videos are intended to help explain the existing WCAG 2 requirements (SCs and guidelines). That is, their scope is limited to these specific WCAG 2 requirements unless we decide to change the requirements.

Also, most videos **will** use words from the handle and from the SC itself, in particular relevant terminology. We will avoid using the specific SC number and verbatim phrasing of the handle and SC text to make the video as reusable as possible. However, we cannot at this point guarantee their relevance outside of WCAG 2. Again, unless we reopen the requirements definition.

Finally, during the development process, it is very helpful to use the SC and GL numbers for directory and file names (for scripts, props, etc.). I think the file names (and URIs) of the final/published videos is a WAI website policy question/decision and outside the scope of this project to define.

**Comment by shawna-slh** [comment]
I agree that requirements are set and no need to re-open them. Given the timing of WCAG 2 and 3, and this video project, we want to stick with focus on WCAG 2.

At the same time, the fundamental concepts of digital accessibility don't change much. Therefore, the concepts in (most if not all of) _these videos will apply beyond WCAG 2 now and in the future_.

And indeed, the video scripts that I've seen, seem like they surely will apply to WCAG 3 -- even if not necessarily map to specific "outcomes".

> We will avoid using the specific SC number and verbatim phrasing of the handle and SC text to make the video as reusable as possible.

I disagree (and am open to being convinced otherwise). I actually think that it would be **good** to use “verbatim phrasing” when and where it is clear wording for the point we’re getting across in the video. It would be good to reinforce WCAG 2 wording and have clear relationship. I don’t see that that makes it less relevant in other contexts.

I also hope that WCAG 3 uses WCAG 2 wording wherever it fits well in WCAG 3.
 
+1 to using SC and GL numbers during development

<hr>

**Draft statement:**
These videos specifically address WCAG 2 guidelines and success criteria (SC). The videos will not include the guideline or SC number, to make them more useful in other contexts, such as WCAG 3 in the future.

We expect that most of these videos will be directly relevant to WCAG 3. (Although, there will not be a one-to-one relationship between the videos and the specific WCAG 3 “outcomes”.)

_Comments?_

**Comment by nitedog** [comment]
Subtle suggestion:

> These videos specifically address WCAG 2 guidelines and success criteria (SC). The videos will not include the guideline or SC number, to make them more useful in other contexts, such as WCAG 3 in the future.
> 
> We expect that most of these videos will be directly relevant to WCAG 3. (Although, there **might** not be a one-to-one relationship between the videos and the specific WCAG 3 “outcomes”.)

That is, change "will not be a one-to-one relationship" to "might not be a one-to-one relationship"

**Comment by shawna-slh** [comment]
Given the way things are looking with WCAG 3, there is **essentially 0 chance**.

I'd prefer to keep it simple and say: "...there will not be a one-to-one relationship..."

However, if you feel strongly against that, here's another idea -- "...there **will likely not** be a one-to-one relationship...":

> These videos specifically address WCAG 2 guidelines and success criteria (SC). The videos will not include the guideline or SC number, to make them more useful in other contexts, such as WCAG 3 in the future.
>
>We expect that most of these videos will be directly relevant to WCAG 3. (Although, there will likely not be a one-to-one relationship between the videos and the specific WCAG 3 “outcomes”.)

**Comment by nitedog** [comment]
I'm fine with the first part of the statement and feel it is sufficient. I cannot speak to the second part given the early stage of WCAG 3. In fact, to me the two sentences in that second part of the statement contradict each other to some degree. Is it important for us to make such predictions (one way or the other) at this stage? In any case, I will go with the consensus on this as long as the first part remains.

**Comment by shawna-slh** [comment]
> Is it important for us to make such predictions (one way or the other) at this stage?

As I mentioned, I keep getting questions about this. I would like to provide such info to limit questions and have a quick, easy place to point to when I do get them.

I am happy to talk through this more with you or anyone who would like. (I feel like this text exchange is not the best way to get in synch.)
